### ISSUE TITLE
Readvars - allow user to select more than 250 variables (CR #5951)

### ISSUE THREAD

**Issue by axelstudios** [NREL/EnergyPlus#849]
## 
#### Description

MJW 27Aug2003
A user at the training course uses something other than Excel to open csv files and it is capable of reading more than Excel's limit of 250+.  Would it be possible to add a user switch to turn off the 250 limit in ReadVars?

LKL 30Aug2003
I went a bit beyond this -- all versions now automatically can go more than 255 variables.  A warning shows up if you go beyond 255 that it might not be readable by Excel.  When I tested this under VF (Lahey is the compiler used for releases), at some point beyond 3500 variables, it got a stack overflow.  So, there is another warning issued there.

Basically, it now follows the style of EnergyPlus and allocates up to the maximum number of variables the person has chosen.

Also, the annoying trailing , on the lines of values is now gone.
## 

External Ref: Advanced workshop, June 2003
Addressed in `03.09.02 V1.1.1.009` by @lklawrie
Last build tested: `03.09.02 V1.1.1.009`
